package.json

```diff
@@ -12,6 +12,7 @@
   },
   "scripts": {
     "test": "yarn build && jest",
+    "detectOpenHandles": "yarn build && jest --detectOpenHandles",
```

Comment:
probably don't need to check this particular test command in

Reply:
Dustin added this to combat a warning we were getting. do you want it removed definitely?

Reply:
```
A worker process has failed to exit gracefully and has been force exited. This is likely caused by tests leaking due to improper teardown. Try running with --detectOpenHandles to find leaks.
```
that is a bug we need to file in the backlog
4:34
make a note that yarn build && jest --detectOpenHandles is involved
4:34
and then remove that from package.json